### PR TITLE
Add expirator build jobs

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- expirator build jobs

**Related issue(s)**

### CODE REVIEW

**Comment by piotrmiskiewicz** [comment]
waiting for a source code of expirator

File: prow/jobs/kyma-project/kyma-environment-broker/kyma-environment-broker-build.yaml [hunk 1]
```diff
@@ -194,6 +194,54 @@ presubmits: # runs on PRs
           - name: signify-secret
             secret:
               secretName: signify-dev-secret
+    - name: pull-keb-expirator-build
+      annotations:
+        description: "Expires instances for given plan and expiration period, build on PR"
```
Comment (jaroslaw-pieszka) [inline prow/jobs/kyma-project/kyma-environment-broker/kyma-environment-broker-build.yaml:199]:
```suggestion
        description: "Expires instances for given plan and expiration period, built on PR"
```

File: prow/jobs/kyma-project/kyma-environment-broker/kyma-environment-broker-build.yaml [hunk 2]
```diff
@@ -594,6 +642,57 @@ postsubmits: # runs on main
           - name: signify-secret
             secret:
               secretName: signify-dev-secret
+    - name: post-keb-expirator-release-build
+      annotations:
+        description: "Expires instances for given plan and expiration period, build on tag creation"
```
Comment (jaroslaw-pieszka) [inline prow/jobs/kyma-project/kyma-environment-broker/kyma-environment-broker-build.yaml:647]:
```suggestion
        description: "Expires instances for given plan and expiration period, built on tag creation"
```

File: templates/data/kyma-environment-broker-build.yaml [hunk 3]
```diff
@@ -155,6 +155,44 @@ templates:
                   global:
                     - kaniko_buildpack
                     - jobConfig_postsubmit
+              - jobConfig:
+                  name: pull-keb-expirator-build
+                  image: "europe-docker.pkg.dev/kyma-project/prod/image-builder:v20240327-023631f7"
+                  annotations:
+                    owner: "gopher"
+                    description: "Expires instances for given plan and expiration period, build on PR"
```
Comment (jaroslaw-pieszka) [inline templates/data/kyma-environment-broker-build.yaml:163]:
```suggestion
                    description: "Expires instances for given plan and expiration period, built on PR"
```

File: templates/data/kyma-environment-broker-build.yaml [hunk 4]
```diff
@@ -155,6 +155,44 @@ templates:
...
+                inheritedConfigs:
+                  global:
+                    - kaniko_buildpack
+                    - jobConfig_presubmit
+              - jobConfig:
+                  name: post-keb-expirator-release-build
+                  image: "europe-docker.pkg.dev/kyma-project/prod/image-builder:v20240327-023631f7"
+                  annotations:
+                    owner: "gopher"
+                    description: "Expires instances for given plan and expiration period, build on tag creation"
```
Comment (jaroslaw-pieszka) [inline templates/data/kyma-environment-broker-build.yaml:180]:
```suggestion
                    description: "Expires instances for given plan and expiration period, built on tag creation"
```

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**